test/test_inference1/config.yml

```diff
@@ -0,0 +1,200 @@
+name: test_inference1
```

Comment:
it would be good if you could meet with @shauntruelove to see if this config needs to be updated to something more modernish